packages/block-library/src/query-loop/style.scss

```diff
@@ -0,0 +1,17 @@
+.wp-block-query-loop {
+	max-width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+
+	&.is-grid {
+		flex-direction: row;
+	}
+	@include break-small {
+		@for $i from 2 through 6 {
+			&.is-grid.columns-#{ $i } > div {
+				width: calc((100% / #{$i}) - 1em);
```

Comment:
Why the `- 1em` ?

Reply:
>I will probably need suggestions for the styling(css) so any input will be helpful.

I took that code from you actually :) from `rss` but I haven't gone deep with css requirements yet.
I believe there should be a way to have some margins or something to be able to apply border as well etc...

I'll need to read more about `flex` and any suggestion is welcome.